react/components/Button/index.js

```diff
@@ -123,7 +124,11 @@ class Button extends Component {
     }
 
     if (block) {
-      classes += 'w-100 h-100 '
+      classes += 'w-100 '
+    }
+
+    if (fullHeight) {
```

Comment:
`if (block || fullHeight) {}`?

Reply:
different classes. but me and @lbebber thought it's better to remove this fullHeight feature. The problem i was fixing was caused by other issue.